docs/guide/ecosystem/documentation.md

```diff
@@ -1,91 +1,227 @@
+     */
+    colorExtends: {
+      type: String,
+      default: '#333'
+    }
+  }
+```
+
+#### Pros
+
+- Displays a live editor to try components
+- Meant for design system creation (powers vue-design-system)
+- A strong React community
+
+#### Cons
+
+- Little configurability on output (HTML only)
+- JSDoc can be verbose
+- Scaffolding built in React
 
 ### Vuese
+
 https://github.com/vuese/vuese
 
-### Vuedoc
-https://gitlab.com/vuedoc/md
+Vuese is a documentation oriented.
+It sacrifices a bit of user experience for a lot of integrability.
+If you have to generate your doc to integrate it, this is probably the tool that you want.
 
-### JSDoc Vuejs
-https://github.com/Kocal/jsdoc-vuejs
+Example: https://buptsteve.github.io/markdown-it-vuese/example/demo.html#props
+
+It supports all the following vue features:
+
+- Single file components
+- Class style components
+- Template and script commenting (Events, Methods, Props, Slots)
+- JSX & TSX
+- TypeScript & Flow
+- Vuex & Vue-i18n
+
+#### Documentation style
+
+vuese uses its own documentaion format more compact than JSDoc
+
+```javascript
+props: {
+  // The name of the form
+  name: {
+    type: String
+  }
+}
+```
+
+#### Pros
+
+- Compact documentation
+- Vuepress default output
+- Configurable docute or markdown outputs
+- Entirely made with VueJs
+
+#### Cons
+
+- Static code samples
 
 ### Vue Design System
+
 https://github.com/viljamis/vue-design-system
 
-## Other useful tools
+Vue designs system is a combination of vue-styleguidist and [salesforce-ux/theo](https://github.com/salesforce-ux/theo) that helps teams build a starter kit for their design system. It is highly opinionated. Choose it if you are starting a new project or if you don't want to think about your own naming.
 
-### Storybook
-[Storybook](https://storybook.js.org/) is a development environment for UI components. It allows developers to setup interactive examples of how a UI component can be used. Simply put, you can setup a live demo of a Vue component by passing different props to it, creating a working sandbox.
+vueds documentation is a great read, even if you are not going to use it.
 
-This is very powerful when working in big teams with lots of custom UI components. A real rendered component with usage example can be easier to understand than just a text example.
+## Documentation Generators
 
-The project is very mature, used by Angular, React and many other big players.
+Automatic documentation generators scan your files and output a structured format, that is easy to either use directly or be passed to another tool.
 
-#### Useful Links
-* [Official Guide](https://storybook.js.org/basics/guide-vue/)
-* [Using Storybook with VueJS](https://auth0.com/blog/using-storybook-with-vuejs/)
-* [Visual testing for Storybook for Vue.js](https://docs.percy.io/docs/storybook-for-vue)
+Such tools can scan your files for JSDoc blocks and output either a json file or full blown websites, ready to deploy.
```

Comment:
We have some repetition here with the above line. 
Also how is Vuese different than Vuedoc? Doesnt it generate just plain static markdown file too?